snap/snapcraft.yaml

```diff
@@ -95,6 +63,7 @@ apps:
     extensions: [gnome]
     desktop: usr/share/applications/element-desktop.desktop
     command: launcher
+    autostart: element-desktop.desktop
```

Comment:
Can you please separate the autostart change into a separate PR?

Reply:
Will it be okay, if I just edit the commit message?

Reply:
I'd prefer a separate PR. In case we want to revert that if we have negative feedback, it would be a simple revert.

Reply:
Done, but I don't think that'd break anything.

Reply:
It won't break, but users might not want it to autostart by default. I want to consider that a bit before we merge that change.

Reply:
No no, it supports the autostart feature. The app will not autostart by default. When users will enable the autostart on login option in the app, it'll place that desktop file under .config/autostart. If we recognize it as a valid autostrat file from here, snapd will allow the autostart.

https://snapcraft.io/docs/snapcraft-yaml-schema#p-21225-app-directives